Utils/release_notes_generator.py

```diff
@@ -248,13 +252,18 @@ def get_release_notes_dict(release_notes_files):
     return release_notes_dict, packs_metadata_dict
 
 
-def merge_version_blocks(pack_name: str, pack_versions_dict: dict, pack_metadata: dict):
+def merge_version_blocks(pack_name: str, pack_versions_dict: dict, pack_metadata: dict, wrap_pack: bool = True,
+                         add_whitespaces: bool = True, wrapper: str = ''):
     """
     merge several pack release note versions into a single block.
 
     Args:
         pack_name: pack name
         pack_versions_dict: a mapping from a pack version to a release notes file content.
+        wrap_pack: whether to wrap the rn with the pack header or not.
+        add_whitespaces: a parameter to pass to construct_entities_block function which indicates
+        whether to add whitespaces to the entity name or not
+        wrapper: a wrapper to wrap the outputs
```

Comment:
not too great arg names, `wrap_pack` and `wrapper` and not too clear from the docstring what does `wrapper` actually do